### PR TITLE
Localisation updates from https://translatewiki.net.

### PR DESCRIPTION
**Pull request by niedzielski**

### CODE REVIEW

File: Wikipedia/Localizations/ksh.lproj/Localizable.strings [hunk 1]
```diff
@@ -151,37 +171,43 @@
...
+"about-testers-details" = "Qualliteit jepröhf vun $1";
 "about-translators" = "Övversäzer";
 "about-translators-details" = "Övvesaz vun Freihwelleje op $1";
-// Fuzzy
 "about-libraries" = "Jebruchte Projrammbibblijothke";
 "about-libraries-license" = "Lezänz";
 "about-libraries-licenses" = "Lėzänze";
 "about-content-license" = "De Lezänz för der Ennhalld";
+"about-content-license-details" = "Wann nit annders aanjejovve, de Ennhallde sin ze hann onger ene $1.";
+"about-content-license-details-share-alike-license" = "<i lang=\"en\">Creative Commons</i> Lezänz, et Wiggerjävve es zohjelohße onger dersellve Bedengunge";
```
Comment (montehurd) [inline Wikipedia/Localizations/ksh.lproj/Localizable.strings]: Oh no! The test should have caught this html... hmm...

**Comment by montehurd** [comment]
I restarted the tests since Brian's patch was merged... fingers crossed the test fails properly this time...

**Comment by montehurd** [comment]
Hmmm still passing when it shouldn't :-1: 

@coreyfloyd I need some help tracking down why this PR passes all tests when it fails to do so locally... is weird!

**Comment by coreyfloyd** [comment]
@montehurd - will need to wait until I am on land before I can look at this. VPNs do not like airplane internet.

I'll check the logs and see whats happening.

**Comment by coreyfloyd** [comment]
@montehurd this is supposed to be failing correct?

**Pull request merged**